releases/2022.08.1.yaml

```
@@ -0,0 +1,237 @@
+    version: 2020.2
+  CppAD:
+    type: git
+    url: https://github.com/coin-or/CppAD.git
+    version: 20220000.4
+  casadi:
+    type: git
+    url: https://github.com/ami-iit/casadi.git
+    version: 3.5.5.3
+  YCM:
+    type: git
+    url: https://github.com/robotology/ycm.git
+    version: v0.14.2
+  YARP:
+    type: git
+    url: https://github.com/robotology/yarp.git
+    version: v3.7.2
+  ICUB:
+    type: git
+    url: https://github.com/robotology/icub-main.git
+    version: v1.27.0
+  ICUBcontrib:
+    type: git
+    url: https://github.com/robotology/icub-contrib-common.git
+    version: v1.19.0
+  robots-configuration:
+    type: git
+    url: https://github.com/robotology/robots-configuration.git
+    version: v1.26.1
+  GazeboYARPPlugins:
+    type: git
+    url: https://github.com/robotology/gazebo-yarp-plugins.git
+    version: v4.5.1
+  icub-models:
+    type: git
+    url: https://github.com/robotology/icub-models.git
+    version: v1.25.0
+  yarp-matlab-bindings:
+    type: git
+    url: https://github.com/robotology/yarp-matlab-bindings.git
+    version: v3.7.3
+  RobotTestingFramework:
+    type: git
+    url: https://github.com/robotology/robot-testing-framework.git
+    version: v2.0.1
+  icub-tests:
+    type: git
+    url: https://github.com/robotology/icub-tests.git
+    version: v1.24.0
+  blocktestcore:
+    type: git
+    url: https://github.com/robotology/blocktest.git
+    version: v2.3.5
+  blocktest-yarp-plugins:
+    type: git
+    url: https://github.com/robotology/blocktest-yarp-plugins.git
+    version: v1.1.3
+  iDynTree:
+    type: git
+    url: https://github.com/robotology/idyntree.git
+    version: v7.0.0
+  BlockFactory:
+    type: git
+    url: https://github.com/robotology/blockfactory.git
+    version: v0.8.3
+  WBToolbox:
+    type: git
+    url: https://github.com/robotology/wb-toolbox.git
+    version: v5.5.0
+  OsqpEigen:
+    type: git
+    url: https://github.com/robotology/osqp-eigen.git
+    version: v0.7.0
+  UnicyclePlanner:
+    type: git
+    url: https://github.com/robotology/unicycle-footstep-planner.git
+    version: v0.4.1
+  walking-controllers:
+    type: git
+    url: https://github.com/robotology/walking-controllers.git
+    version: v0.5.1
+  icub-gazebo-wholebody:
+    type: git
+    url: https://github.com/robotology/icub-gazebo-wholebody.git
+    version: v0.1.0
+  whole-body-controllers:
+    type: git
+    url: https://github.com/robotology/whole-body-controllers.git
+    version: v2.5.6
+  whole-body-estimators:
+    type: git
+    url: https://github.com/robotology/whole-body-estimators.git
+    version: v0.9.0
+  walking-teleoperation:
+    type: git
+    url: https://github.com/robotology/walking-teleoperation.git
+    version: v1.3.1
+  yarp-devices-forcetorque:
+    type: git
+    url: https://github.com/robotology/yarp-devices-forcetorque.git
+    version: v0.3.0
+  wearables:
+    type: git
+    url: https://github.com/robotology/wearables.git
+    version: v1.4.0
```

```suggestion
    version: v1.5.0
```

We could use the just-released v1.5.0 version if https://github.com/robotology/robotology-superbuild/pull/1259 pass the checks

---

Note that this has branched from `releases/2022.08` branch, so even if we use `v1.5.0` we will not get the python bindings. We could backport https://github.com/robotology/robotology-superbuild/pull/1259 to `releases/2022.08`, but unless it is strictly necessary I would try to avoid that.

---

Ok, got it.

Just to double-check, @GiulioRomualdi @S-Dafarra what do you think. Since I think recent activities require wearable pyhton bindings, would it be ok to switch the flag `WEARABLES_COMPILE_PYTHON_BINDINGS` manually?

---

For what concerns the iFeel walking, the plan would be to switch to C++ soon, so it should not be a big deal to switch that setting manually for the time being.

---

Anyhow, we can bump iFeel to v1.5.0 also here, no problem.

---

Done!